src/main/kotlin/no/nav/familie/ef/mottak/mapper/ArkiverDokumentRequestMapper.kt

```diff
@@ -38,6 +89,14 @@ object ArkiverDokumentRequestMapper {
                         dokumenttype = dokumenttypeVedlegg)
     }
 
+    private fun tilEttersendingDokument(vedlegg: EttersendingVedlegg, dokumenttypeVedlegg: Dokumenttype): Dokument {
```

Comment:
Jeg synes det er litt vanskelig å skjønne forskjellen mellom vedlegg og ettersendingsvedlegg

Reply:
To forskjellige objekter - som tilfeldigvis har samme struktur...

Reply:
Men bittelitt forskjellige